packages/amplication-client/src/Components/ErrorMessage.tsx

```diff
@@ -0,0 +1,18 @@
+import {Icon} from "@rmwc/icon";
+import React from "react";
+
+type Props = {
+    error: string | string[] | undefined | null;
+    className: string;
+}
+
+export const LoginErrorMessage = ({error, className}: Props) => {
+    return (
+        <div>
+            {error && (<div className={`${className}__login-error`}>
```

Comment:
Please add a unique class name for the component (e.g. amp-error-message)
the prop className should only be used to extend the styling but the component should come with its win style.